intro/functions/code.js

```diff
@@ -1,2 +1,14 @@
 module.exports = {
+  add: function(a, b) {
```

Comment:
Reimplement using explicit exports (no module.exports). And try using lambda style function definitions vs traditional function definitions.